package.json

```diff
@@ -23,9 +23,16 @@
     "@angular/platform-server": "^17.3.0",
     "@angular/router": "^17.3.0",
     "@angular/ssr": "^17.3.3",
+    "@stomp/stompjs": "^7.0.0",
+    "@types/socket.io": "^3.0.2",
+    "@types/stompjs": "^2.3.9",
+    "angular-sockjs": "^0.1.0",
```

Comment:
This library is not needed in the application, I forgot to uninstall it when we were testing with @cowboy.